Source/Core/Core/CoreTiming.cpp

```diff
@@ -380,6 +380,7 @@ void Idle()
     Fifo::FlushGpu();
   }
 
+  PowerPC::UpdatePerformanceMonitor(PowerPC::ppcState.downcount, 0, 0);
```

Comment:
Is it fine to use 0 as the number of load/store instructions? Idle loops typically have at least one load, right? (Not sure if this actually matters in practice.)

Reply:
Technically it's wrong, but it's not trivial to fix, and the behavior of the load/store counter is the same as before this line was added.

Reply:
Yeah, I guess counting the number of load/store instructions in PPCAnalyzer would only give an approximation anyway...